tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
Yeah probably. Talk to @memontic-msft about that.

---
In reply to: [108996733](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108996733) [](ancestors = 108996733,108995360,108995058,108994710,108994382,108993288)